app/assets/stylesheets/application.css

```diff
@@ -252,7 +247,7 @@ a:hover {
 }
```

Comment:
Seems like this is not being used. Please add a comment next time if it is necessary for something that you would like to add later.